2/contrib/openshift/configuration/init.groovy.d/setup_wizard_disable.groovy

```diff
@@ -0,0 +1,9 @@
+#!groovy
+
+import jenkins.model.*
+import hudson.util.*;
+import jenkins.install.*;
+
+def instance = Jenkins.getInstance()
+
+instance.setInstallState(InstallState.INITIAL_SETUP_COMPLETED)
```

Comment:
Is this needed, if we have the system property, is it suffisant ?

Reply:
System property is sufficient, I tested it.